app/views/decks/_deck_summary.html.erb

```diff
@@ -0,0 +1,106 @@
+                    </a>
+                    <a href="#">
+                      <li class="list-item-label"><span class="label label-free"><%= resource.where(tier: 'free').count %> Free</span></li>
+                    </a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td><%= @categories.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <% @categories.each do |category| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= @category_deck_links.where(category_id: category.id).count %> <%= category.name %></span>
+                      </li>
+                    <% end %>
+                  </ul>
+                </td>
+              </tr>
+
+            <% elsif resource == @cards %>
+              
+              <tr>
+                <th><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span>Created At</th>
+                <td>3/20/2016</td>
+                <td></td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-book" aria-hidden="true"></span>Cards</th>
+                <td><%= resource.count %></td>
+                <td>
+                  <ul class="list-inline">
+                    <a href="#"><li class="list-item-label"><span class="label label-needs-review">0 Needs Review</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-published">0 Published</span></li></a>
+                    <a href="#"><li class="list-item-label"><span class="label label-unpublished">0 Unpublished</span></li></a>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-bookmark" aria-hidden="true"></span>Categories</th>
+                <td>2</td>
+                <td>
+                  <ul class="list-inline">
+                    <li class="list-item-label"><span class="label label-default">25 Category One</span></li>
+                    <li class="list-item-label"><span class="label label-default">25 Category Two</span></li>
+                  </ul>
+                </td>
+              </tr>
+              <tr>
+                <th><span class="glyphicon glyphicon-tags" aria-hidden="true"></span>Question Type</th>
+                <td></td>
+                <td>
+                  <ul class="list-inline">
+                    <% @card_types.each do |type| %>
+                      <li class="list-item-label">
+                        <span class="label label-default"><%= resource.where(card_type: type[1]).count %> <%= type[0].humanize %></span>
+                      </li>
```

Comment:
we may not need the `@card_types` instance var here and just use `Card.card_types` (which would also mean removing it from the `deck_controller`. you can pass both the key and the value to the `do` block like so

```ruby
Card.card_types.each do |key,value|
  resource.where(card_type: value).count
  ...
  key.humanize
end
```